diff --git a/src/meson.build b/src/meson.build
index de98cc4..e2fe6a6 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -48,7 +48,6 @@ switchboard_dep = dependency('switchboard-2.0')
 switchboard_plugsdir = switchboard_dep.get_pkgconfig_variable('plugsdir', define_variable: ['libdir', libdir])
 
 gnome_keyboard_dep = dependency('libgnomekbd')
-gnome_keyboard_ui_dep = meson.get_compiler('c').find_library('libgnomekbdui')
 
 ibus_dep = dependency('ibus-1.0')
 if(ibus_dep.version().version_compare('>=1.5.19'))
@@ -69,7 +68,6 @@ shared_module(
         dependency('libhandy-1', version: '>= 0.90.0'),
         dependency('libxml-2.0'),
         gnome_keyboard_dep,
-        gnome_keyboard_ui_dep,
         ibus_dep,
         switchboard_dep
     ],
